src/components/Conference.tsx

```diff
@@ -142,15 +142,18 @@ export class Conference extends React.Component<IConferenceProps, IConferenceSta
 
     private renderStream(stream: ConferenceStream) {
         let className = 'rcw-remote-stream';
+        let muted;
         if (stream.local) {
```

Comment:
why do we need to hard code this? 
<Stream stream={localStream.stream} muted={true} /> => Local Stream should be muted by this, right?